code/datums/keybinding/yautja.dm

```diff
@@ -415,3 +415,20 @@
 	if(istype(held_item))
 		held_item.fold_combistick()
 	return TRUE
+
+/datum/keybinding/yautja/guard_gloves
+	hotkey_keys = list("Space")
```

Comment:
pick a different key or unbind this by default, otherwise every player gets a message on join due to the keybind conflict with unique-action